src/index.tsx

```diff
@@ -1,6 +1,11 @@
 import * as React from "react";
 import { findDOMNode } from "react-dom";
 
+export interface IPrintContextProps {
+    handlePrint: () => void,
+};
+export const PrintContext = React.createContext({} as IPrintContextProps);
```

Comment:
Is there a way to make this conditional based on `React.createContext` existing? Right now we support React v15 which does not have `createContext` (introduced in React `16.3.0`). I would actually be ok making a major version change to drop React 15, however, I'd want to get a few more breaking changes in I've been meaning to do, which would delay shipping this by a few days.